### PR TITLE
added new acis packages bep_pcb_check fep1_actel_check fep1_mong_check

### PR DESCRIPTION
**Pull request by javierggt**
Changes are still not finished.

### CODE REVIEW

**Comment by jzuhone** [comment]
I still need some clarification from @jeanconn as to whether or not we need to edit the other ACIS packages similarly.

File: pkg_defs/bep_pcb_check/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+package:
+  name: bep_pcb_check
+  version:  {{ SKA_PKG_VERSION }}
+
+build:
+  noarch: python
+  script_env:
+    - SKA_TOP_SRC_DIR
```
Comment (jeanconn) [inline pkg_defs/bep_pcb_check/meta.yaml:8]: I think this is not required.

**Comment by jeanconn** [comment]
I'm fine with that being up to @javierggt .  I think we're just talking about recipe improvements (so probably don't *need* the updates but not sure when it is most convenient to get them in).

**Comment by jeanconn** [comment]
And @javierggt did you just make this to supersede #323 ?

**Comment by javierggt** [comment]
Oops, no, I did not intend to superseed that one. Sorry about that. I think I will just close this one.

**Comment by jeanconn** [comment]
The tiny issue is that we can only make suggestions to John's PR.  So if you just want to get it in with these bits that use `python setup.py install` that's fine.

**Comment by javierggt** [comment]
I wonder if the dependencies in this PR are better. I created them by using findimports. And the build dependencies should be a rather short list.

File: pkg_defs/bep_pcb_check/meta.yaml (hunk 1 above)
Comment (javierggt) [reply pkg_defs/bep_pcb_check/meta.yaml]: Right. I removed them.

**Comment by javierggt** [comment]
So... I decided to go with this one in the end. The main difference is the requirements, I believe.

I am listing only explicit requirements:
- numpy is not a requirement because it is not imported explicitly. matplotlib is imported explicitly.
- six is not imported by any of these
- added ska_helpers and testr to all because they are imported.

**Comment by jeanconn** [comment]
I'm fine with only explicit requirements.  In the end it doesn't buy much (these require acis_thermal_check which has numpy as dependence) but doesn't cause any problems.

**Comment by jzuhone** [comment]
@javierggt happy for this to be the PR. My only question is if the other ACIS packages (`dpa_check`, `dea_check`, `psmc_check`,`acisfp_check`) should have a similar setup, because right now they do not.

**Comment by javierggt** [comment]
@jzuhone, I think it is not necessary at this time (for this coming release). In the future, that will certainly be done.

**Comment by javierggt** [comment]
We need to close this before closing sot/skare3/pull/342

**Comment by jeanconn** [comment]
@jzuhone I've just got really simple run tests of the other checkers in our test suite.  Did you want to suggest similar run/smoke tests for us to run for the new checkers?

**Comment by jzuhone** [comment]
@jeanconn did you want to try the new answer testing? it should just run with e.g.:

```python
import psmc_check
psmc_check.test()
```

**Comment by jeanconn** [comment]
Yes!  I think we probably want both?  In that we have the "run like they do for load review" stuff like

https://github.com/sot/ska_testr/blob/8092f9a5a7ad2036fcc962006602def7796f12da/packages/acisfp_check/test_regress.sh

but should also add the new answer testing?  Or do you think the new answer testing trumps the "run like they do for load review" tests?  Doesn't belong in this PR anyway but...

**Comment by jzuhone** [comment]
Probably do both, to be honest.

**Pull request merged**